amida-tech/amida-react-skeleton-graphql
# Feature/foundational improvements

### Related tickets:
- n/a

### What's changing? Any breaking changes?
- Add SASS via `node-sass`
  - Went ahead and refactored existing `.css` into `.scss` (with BEM formatting where necessary)
  - TODO: While I added some variables for `Nav`, we should reference them elsewhere (`/styles`?)
- Added Layout+Nav Components with styling and tests

### Manual test cases?
- First run `yarn` to get the new packages
- Run `yarn test`, ensure all tests pass
- Run `yarn start` and check out the `Nav` bar and `Menu`, test the `Link`s in the pages, in the menu, etc

### Any additional context/background?
- n/a

### Screenshots (if appropriate):
- n/a
